test_files/sickle/static.ts

```diff
@@ -0,0 +1,11 @@
+class Static {
+  // This should not become a stub declaration.
+  static bar = 3;constructor() {
```

Comment:
ew, glad you check in those files, so we can see the ugly output of sickle :) I wonder if it is worth producing better looking code at some point. (no action required).